rosdep/base.yaml

```diff
@@ -5381,7 +5381,9 @@ pybind11-dev:
 python-dev:
   debian: [python-dev]
   fedora: [python2-devel]
-  ubuntu: [python-dev]
+  ubuntu:
+    '*': [python-dev]
+    'focal': [python-dev-is-python2]
```

Comment:
I think the issue you're seeing is because `python-dev` does not exist in Focal. Maybe the correct thing to do here is install [`python2-dev`](https://packages.ubuntu.com/focal/python2-dev) instead?

Reply:
python-dev is 'pure virtual', though I'm not super familiar with what that means. I think you're right though that I should just use python2-dev.

Should I create a new rosdep key for that, or should I replace the focal entry for `python-dev` so as to keep the package.xml consistent?

Reply:
Replacing the focal entry as you've done seems right to me. Let's get a third opinion in any case.